### ISSUE TITLE
Add Setup Wizard

### ISSUE THREAD

**Issue by cjpais** [holic/web3-scaffold#40]
Something similar to [powerlevel9k](https://github.com/Powerlevel9k/powerlevel9k) where you use a shell script to set up your bash prompt.

I think there are a few config options that would be cool to have in particular, and set up your scaffold in the way you prefer through running the script. It starts as something very vanilla if the script is never run.

This might defeat the purpose of the opinionated scaffold, but gives more flexibility for everyone to use IMO.

**Init**

Set your package names in all the package.json files

**Package Manager**
- [ ] pnpm (default)
- [ ] yarn + lerna?
- [ ] npm + lerna

**Wallet Connect Options**
- [ ] RainbowKit/Wagmi (default)
- [ ] useWallet/web3modal

**GraphQL Options**
- [ ] urql (default)
- [ ] swr + graphql-request

**Editor Options**
(multi-select)
- [ ] prettier (default)
- [ ] eslint (default)

**CSS Options**
- [ ] Tailwind (default)
- [ ] Vanilla
- [ ] etc

Also I'm happy to start on this after I get a few things done

**Comment by frolic** [comment]
Given that we just discovered [create-web3](https://www.create-web3.xyz/) and they're further along this route of customization, I may keep this scaffold opinionated.

**Comment by cjpais** [comment]
Makes sense, feel free to close this issue if you think you'll want to keep it opinionated. 

I hadn't seen the discussion of this yet when created

**Comment by jamiew** [comment]
I'd also vote to keep things opinionated and close this. maybe a note in the README suggesting create-web3?
